serialize/Cargo.toml

```diff
@@ -25,11 +25,15 @@ arrayvec.workspace = true
 digest.workspace = true
 num-bigint.workspace = true
 rayon = { workspace = true, optional = true }
+serde = { workspace = true, optional = true }
+serde-encoded-bytes = { workspace = true, optional = true, features = ["base64"] }
+serde_with = { workspace = true, optional = true, default-features = false }
 
 [dev-dependencies]
 sha2.workspace = true
 sha3.workspace = true
 blake2.workspace = true
+serde_json.workspace = true
```

Comment:
This should not be on by default.

Ideally serde should not even be an optinal dependency, which might inject code everywhere, but only a dependency of a seperate crate.

Reply:
I suppose okay, maybe it could do so only in CI here?

The concerns over serde is first that it shipped a binary that runs during builds once before, and second that at least one of the maintainers supposedly works for Palantir.